intl_pluralrules/README.md

```diff
@@ -14,6 +14,14 @@ Status
 
 Currently produces operands compliant with CLDR 33 into Rust 1.27 and above.
 
+**Using External CLDR Data**
+
+If you would like to use plural rules that are not the specified version above, you can regenerate intl_pluralrules's internal rules with the command:
```

Comment:
I would rename/extend that section - this procedure is primarily going to be used to update the rules.rs file when new CLDR is released, so it should be mentioned as a procedure to update `intl-pluralrules` to the next version of CLDR.